contracts/consensus/Consensus.sol

```diff
@@ -574,7 +656,7 @@ contract Consensus is MasterCopyNonUpgradable, CoreLifetimeEnum, MosaicVersion,
         uint256 _sourceBlockHeight,
         uint256 _targetBlockHeight
     )
-        external
+        public
```

Comment:
Why it is changed to public?

Reply:
Due to the `Stack too deep` error

Reply:
Please create a followup ticket to make it external.

Reply:
Created: https://github.com/mosaicdao/mosaic-1/issues/245